### DISCUSSION TITLE
Support piping external output into helix directly (MacOS?)

### DISCUSSION THREAD

**Discussion by Manosmer** [helix-editor/helix#4158] [Ideas] [answered]
Currently on MacOS, the only way to view any output from an external tool like `git diff` on Helix is by piping the output into a file and then opening it or by using `:insert-output git diff` inside Helix. It would be convenient if there was support for `git diff | hx`, similar to Neovim or Vim's `git diff | vim -`. In the current state I get the following error message:

> $ git diff | hx
> 
> Error: unable to create new application
> 
> Caused by:
>     Piping into helix-term is currently not supported on macOS

In short, my idea is to be able to just run `$git diff | hx` or some other external tool and end up with helix's scratch buffer, populated with that tool's output.

**Comment by Omnikar** [comment, marked as answer]
See https://github.com/helix-editor/helix/pull/996#issuecomment-962518802

**Reply by Manosmer** [reply]
Thank you for the info
